.openpublishing.redirection.json

```
@@ -2232,6 +2232,26 @@
             "source_path": "docs/visual-basic/reference/vb6-support.md",
             "redirect_url": "/visualstudio/vb6/vb6-support",
             "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/framework/data/adonet/ef/language-reference/csdl-specification.md",
+            "redirect_url": "/ef/ef6/modeling/designer/advanced/edmx/csdl-spec",
+            "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/framework/data/adonet/ef/language-reference/csdl-ssdl-and-msl-specifications.md",
+            "redirect_url": "/ef/ef6/index",
```

I'd also move to this one since you can find the other two in here as well

```suggestion
            "redirect_url": "/ef/ef6/modeling/designer/advanced/edmx/csdl-spec",
```